src/test/Pool.t.sol

```diff
@@ -20,7 +20,7 @@ contract PoolTest is BaseTest {
         emit log_named_uint("block.timestamp start", block.timestamp);
 
         dai = new Dai();
-        pool = new NFTPool(CYCLE_SECS, dai);
+        pool = new NFTPool(CYCLE_SECS, IDai(address(dai)));
```

Comment:
In `src/test/TestDai.sol` we should have `contract Dai is IDai {`, it'll be more precise and will prevent need for things like this.

Reply:
Done

Reply:
Ok, now we can remove these castings, right?